bigquery/simpleapp/simpleapp.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command simpleapp queries the Shakespeare sample dataset in Google BigQuery.
+// [START bigquery_simple_app_all]
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"cloud.google.com/go/bigquery"
+	"google.golang.org/api/iterator"
+
+	"golang.org/x/net/context"
+)
+
+func main() {
+	proj := os.Getenv("GCLOUD_PROJECT")
+	if proj == "" {
+		fmt.Println("GCLOUD_PROJECT environment variable must be set.")
+		os.Exit(1)
+	}
+
+	rows, err := Query(proj)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	err = PrintResults(os.Stdout, rows)
+	if err != nil {
+		log.Fatalln(err)
+	}
+}
+
+// Query returns a slice of the results of a query.
+// [START bigquery_simple_app_query]
+func Query(proj string) (*bigquery.RowIterator, error) {
+	ctx := context.Background()
+
+	client, err := bigquery.NewClient(ctx, proj)
+	if err != nil {
+		return nil, err
+	}
+
+	query := client.Query(
+		"SELECT " +
```

Comment:
Done. Though I did have to concat in the dataset name, since standard SQL syntax uses backticks for table names.